docs/appendices/release-notes/5.7.2.rst

```diff
@@ -104,7 +104,7 @@ Fixes
   query.
 
 - Fixed an issue that could lead to out-of-memory errors when executing
-  ``lead`` or ``lag`` window functions on a large table.
+  window functions on a large table.
```

Comment:
If it makes it to 5.7.2 - lag/lead also have this issue, so common change entry probably makes sense instead of having 2 similar.

Reply:
OK, https://github.com/crate/crate/pull/16137 has been merged, so I will rephrase and and a new entry into 5.7.3 once we have it